docs/setup/upgrade/upgrade-migrations.asciidoc

```diff
@@ -149,56 +149,55 @@ upgrade migrations fail. In {kib} 7.5.1 and later, the incompatible configuratio
-attempt to fix a failed migration. Unlike previous versions, {kib} 7.12.0 and
-later does not require deleting any indices to release a failed migration lock.
+to fix a failed migration. Unlike previous versions, {kib} 7.12.0 and
+later does not require deleting indices to release a failed migration lock.
 
-If upgrade migrations fail repeatedly, follow the advice in
+If upgrade migrations fail repeatedly, refer to
 <<preventing-migration-failures, preventing migration failures>>.
-Once the root cause for the migration failure has been addressed,
-{kib} will automatically retry the migration without any further intervention.
-If you're unable to resolve a failed migration following these steps, please contact support.
+When you address the root cause for the migration failure,
+{kib} automatically retries the migration.
+If you're unable to resolve a failed migration, contact Support.
 
 [float]
 [[upgrade-migrations-rolling-back]]
 ==== Rolling back to a previous version of {kib}
 
-If you've followed the advice in <<preventing-migration-failures, preventing migration failures>>
-and <<resolve-migrations-failures, resolving migration failures>> and
-If {kib} is still unable to upgrade successfully, rollback {kib} until
+If you've followed <<preventing-migration-failures, preventing migration failures>>
+and <<resolve-migrations-failures, resolving migration failures>>, and
+{kib} is still unable to successfully upgrade, rollback {kib} until
 you're able to identify and fix the root cause.
 
-WARNING: Before rolling back {kib}, ensure that the version you want to rollback to is compatible with
-your {es} cluster. If the version you're rolling back to is not compatible, you must also rollback {es}.
-Any changes made after an upgrade are lost when rolling back to a previous version.
+WARNING: Before you roll back {kib}, ensure that the version you want to rollback to is compatible with
+your {es} cluster. If the version you want to roll back to is not compatible, you must also rollback {es}.
+Any changes made after an upgrade are lost when you roll back to a previous version.
 
-To rollback after a failed upgrade migration, the saved object indices have to be
-rolled back to be compatible with the previous {kib} version.
+To rollback after a failed upgrade migration, you must also rollback the saved object indices to be compatible with the previous {kib} version.
 
 [float]
 ===== Rollback by restoring a backup snapshot
 
 . Before proceeding, {ref}/snapshots-take-snapshot.html[take a snapshot] that contains the `kibana` feature state.
-   Snapshots include this feature state by default.
+   By default, snapshots include the `kibana` feature state.
 . To make sure no {kib} instances are performing an upgrade migration, shut down all {kib} instances.
-. Delete all saved object indices with `DELETE /.kibana*`.
+. To delete all saved object indices, use `DELETE /.kibana*`.
```

Comment:
In this and the following section of instructions, should we call out the DELETE, GET, PUT instructions in a code block as done previously in the doc?